types/editly/index.d.ts

```diff
@@ -0,0 +1,1063 @@
+// Type definitions for editly 0.11
+// Project: https://github.com/mifi/editly
+// Definitions by: Skayo <https://github.com/Skayo>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { fabric } from 'fabric';
+
+/**
+ * Edit and render videos.
+ *
+ * @param config - Config.
+ */
+declare function Editly(config: Editly.Config): Promise<void>;
```

Comment:
isn't config optional with some pros set to default as per convention?
https://github.com/mifi/editly/blob/1bd0d30acc7924b03fc5ea0615642b38343dad95/index.js#L21-L47
what's common usage scenario? (options no options)

Reply:
Well, if you don't specify any config, the `outPath` will be `undefined` which causes an error anyways. So I just thought I'll make it required right away...

Reply:
From the implementation:

```js
const isGif = outPath.toLowerCase().endsWith('.gif');

// ...

assert(outPath, 'Please provide an output path');
```

Oops 😛